### PR TITLE
LAT and LON are corect shows

### PR DESCRIPTION
**Pull request by mrsima**
If you do not like my previous PR this one could be reasonable.
Time and Date are still not correct. I will continue to test it and fix.
You can forget space and code format differences.
Regards

### CODE REVIEW

File: MinimOsd_Extra/protocols/NMEA.h [hunk 1]
```diff
@@ -316,14 +300,15 @@ store to msg.nmea.lat
 
 
 void read_NMEA(){
+    char string[NMEA_BUF_LENGTH];
     while (Serial.available_S()) {
```
Comment (night-ghost) [inline MinimOsd_Extra/protocols/NMEA.h]: you want to eat out all device's memory? do you ever test your changes???

Comment (mrsima) [inline MinimOsd_Extra/protocols/NMEA.h]: As I told before, every time I made change in code, I compile it, flesh it to OSD and test it. How could I know the result in other way? I do not know other way. I do not want to eat out devices memory either, but I do not know even how to do that or avoid that. I am not programmer. I find a way to get some results in code, that's all. You can help and resolve this problem on better way if you wish. Will you?
Regards

Comment (mrsima) [inline MinimOsd_Extra/protocols/NMEA.h]: I am wondering now, did you ever test NMEA.h and changes you made before announce it?

Comment (night-ghost) [inline MinimOsd_Extra/protocols/NMEA.h]: No, I needn't this mode so it developed by request, so I don't test it. I use MAVlink only.

**Comment by night-ghost** [comment]
Ok I'll do some changes in the image and likeness of the proposed

**Comment by night-ghost** [comment]
v937
* all calculations in NMEA mode are doing in float

File: MinimOsd_Extra/protocols/NMEA.h (hunk 1 above)
Comment (mrsima) [inline MinimOsd_Extra/protocols/NMEA.h]: Thank you so much for your work and time. 
I try it for a 3 times, firmware update to OSD with eeprom reset but get only dark screen. Nothing! File: MinimOsd_Extra_Uni.937DV-NMEA-release.hex extracted from downloaded osd_latest.zip ... Sorry

**Pull request closed without merging**